tests/testthat/test-param-estimates.R

```diff
@@ -6,10 +6,6 @@ if (Sys.getenv("METWORX_VERSION") == "" && Sys.getenv("DRONE") != "true") {
 
 setup({
   cleanup()
-  invisible(copy_model_from(yaml_ext(MOD1_PATH), NEW_MOD2, "model from test-param-estimates.R", .directory = "."))
```

Comment:
Is removing these lines related to the requirement here? It looks like general cleanup.

Reply:
No I just noticed they were hanging around and should've been deleted in https://github.com/metrumresearchgroup/rbabylon/pull/152 because they're no longer used.